### DISCUSSION TITLE
Which side does she swivel to?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#797] [Galaxy Trivia] [answered]
👧👂🦪

**Comment by SANKARAMDAS** [comment]
👧

**Comment by peterbaccaro** [comment, marked as answer]
"Girl with a Pearl Earring" is a famous painting created by Dutch artist Johannes Vermeer around 1665. It is regarded as one of Vermeer's masterpieces and is often considered a timeless symbol of Dutch Golden Age painting. The painting is currently housed in the Mauritshuis museum in The Hague, Netherlands.

In the painting the girl is depicted swivelling her head to the **left side**.
